Main.py

```diff
@@ -0,0 +1,163 @@
+    global team_doesnt_got_turn
+    team_got_turn = team_doesnt_got_turn
+    if team_got_turn is white_team:
+        team_doesnt_got_turn = black_team
+    else:
+        team_doesnt_got_turn = white_team
+
+    timer.switch_timers(team_got_turn, team_doesnt_got_turn)
+
+
+def remove_eaten_pieces(white_team, black_team):
+    for piece in white_team.pieces:
+        if piece.is_eaten:
+            white_team.pieces.remove(piece)
+    for piece in black_team.pieces:
+        if piece.is_eaten:
+            black_team.pieces.remove(piece)
+
+
+def update_game_after_move(piece_clicked, black_team, white_team):
+    pygame.mixer.Sound(os.path.join(SOUNDS_PATH, 'pong.wav')).play()
+
+    switch_turn(white_team, black_team)
+
+    piece_clicked.move_counter += 1
+
+    remove_eaten_pieces(white_team, black_team)
+    white_team.update_score()
+    black_team.update_score()
+
+
+def print_board(white_team, black_team):
+    print(white_team)
+    white_team.print_pieces()
+    print(black_team)
+    black_team.print_pieces()
+
+
+def game_loop(white_team: Team, black_team: Team, is_one_player_playing):
+    print_board(white_team, black_team)
+    black_team.timer.pause()
+    running = True
+    piece_clicked = None
+    global team_got_turn
+    global team_doesnt_got_turn
+    team_got_turn = white_team
+    team_doesnt_got_turn = black_team
+    while running:
+
+        if team_got_turn is black_team and is_one_player_playing:
+            piece_moved = bot.move(white_team, black_team)
+            if piece_moved is None:
+                # Bot has nowhere to go, because it's checkmated.
+                print(f'Team won is {team_doesnt_got_turn}')
+                break
+            update_game_after_move(piece_moved, team_got_turn, team_doesnt_got_turn)
+
+            # Check if after bot move, you are checkmated.
+            if is_checkmated(team_got_turn, team_doesnt_got_turn):
+                print(f'Team won is {team_doesnt_got_turn}')
+                break
+        # else:
+        #     piece_moved = bot.move(black_team, white_team)
+        #     if piece_moved is None:
+        #         # Bot has nowhere to go, because it's checkmated.
+        #         break
+        #     move_played(piece_moved, team_got_turn, team_doesnt_got_turn)
+        #
+        #     # Check if after bot move, you are checkmated.
+        #     if is_checkmated(team_got_turn, team_doesnt_got_turn):
+        #         break
+
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                running = False
+
+            elif event.type == pygame.MOUSEBUTTONDOWN:
+                clicked_square = get_square_clicked()
+
+                if clicked_square is None:
+                    continue
+
+                if piece_clicked is None:
+                    piece_clicked = clicked_square.current_piece
+                    if piece_clicked in team_got_turn.pieces:
+                        piece_clicked.color_next_step()
```

Comment:
Add continue here.
And remove else from below, to remove indentation (הזחה)

Reply:
fixed it